exercises/robot-simulator/lib/example.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:robot_simulator/orientation.dart';
+import 'package:robot_simulator/position.dart';
+
+class Robot {
+  Orientation orientation;
+  Position position;
+
+  Robot(this.position, this.orientation);
+
+  void move(String instructions) {
+    instructions.split("").forEach((character) {
+      if (character == "R") _rotateClockwise();
```

Comment:
Suggestion: `if (character == "trigger") _action();` this is a non-returning statement.
I'm assuming your intentions are to stop checking for other "trigger"s as soon as one is found and it's corresponding action is executed.

So if `R` is the current character and `_rotateClockwise()` is the action being performed, the closure still continues to check for "L" and "A".

My suggestion is use a switch statement here as well. A switch statement makes it clear that only a single case is being handled.
An alternative would be to change to form of `if` statements to 
`if (character == "trigger") return _action();`